ambry-frontend/src/main/java/com/github/ambry/frontend/s3/CompleteMultipartUpload.java

```diff
@@ -0,0 +1,44 @@
+/*
+ * Copyright 2023 LinkedIn Corp. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *
+ */
+
+package com.github.ambry.frontend.s3;
+
+import com.fasterxml.jackson.dataformat.xml.annotation.JacksonXmlElementWrapper;
+import com.fasterxml.jackson.dataformat.xml.annotation.JacksonXmlProperty;
+
+
+public class CompleteMultipartUpload {
```

Comment:
Having the POJO classes as separate class seems to clutter things a bit, do you think is makes sense to move the POJO classes into their corresponding "handler" class, or alternatively move them to a separate package?